en/mulesoft/modules/ROOT/pages/ms_composer_flows.adoc

```diff
@@ -339,13 +339,22 @@ To create an If/Else block:
 . Click the plus sign after the trigger or the last action in your flow.
 . Click *Add Action*.
 . Click *If/Else Block* to define the first branch.
-. Enter the requested criteria.
+. Enter the requested criteria. You can specify that `all conditions should be met (AND)`, `any condition can be met (OR)`, or define <<use-custom-logic,custom logic>> to customize the order and grouping of your conditions.
```

Comment:
when the link is live does this read "define use custom logic to..." should define be cut?

Reply:
Maybe "use custom logic" would be better